src/components/ProjectOverview/GPUProjectInventory.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  InventoryItem,
+  InventoryItemLoading,
+  InventoryItemTitle,
+  useK8sWatchResource,
+} from '@openshift-console/dynamic-plugin-sdk';
+import * as React from 'react';
+import { useTranslation } from '../../i18n';
+import { Pod } from '../../resources';
+
+type GPUProjectInventoryProps = {
+  projectName: string;
+};
+
+const GPUProjectInventory = ({ projectName }: GPUProjectInventoryProps) => {
+  const { t } = useTranslation();
+  const [pods, loaded, loadError] = useK8sWatchResource<Pod[]>({
+    groupVersionKind: {
+      kind: 'Pod',
+      version: 'v1',
+    },
+    isList: true,
+    namespace: projectName,
+  });
+
+  let title: React.ReactNode = t('{{count}} GPU pod', { count: pods.length });
```

Comment:
Isn't that missing some kind of filtering?

Reply:
whops, fixed :)